### ISSUE TITLE
python shell "help" message pop in a different buffer? (like pydoc)

### ISSUE THREAD

**Issue by bingtangben** [jorgenschaefer/elpy#1633]
Python inline documentation ("help" or "?" in IPython) seems a little bit annoying. Could elpy pop in a different buffer displaying help messages.

Thanks.

**Comment by galaunay** [comment]
should be doable, I'll give it a try.

But just out of curiosity, what do you find annoying about having inline documentation in the interpreter ?

**Comment by bingtangben** [comment]
Yes, it's quite annoying. I want to keep the python shell buffur clean and inline documentation is not well highlighted.

**Comment by galaunay** [comment]
A bit of update: 
I gave it a try, but filtering out the inline documentation from the interpreter necessitates more re-writing than I was expected.
It may take some times to have something functional.
